internal/action/show.go

```diff
@@ -49,7 +50,7 @@ func showParseArgs(c *cli.Context) context.Context {
 	}
 
 	if c.IsSet("noparsing") {
-		ctx = ctxutil.WithShowParsing(ctx, !c.Bool("noparsing"))
+		_ = config.FromContext(ctx).SetEnv("core.parsing", fmt.Sprintf("%t", !c.Bool("noparsing")))
```

Comment:
How long is `SetEnv` persisted?
I feel like running this in the REPL might be persisted for more than just one command, no?

Reply:
Yes. This is persistet as long as the binary is running. With the REPL this might be tricky to handle properly.